Assets/Scripts/Region/RegionManager.cs

```diff
@@ -115,6 +115,11 @@ public bool IsInZone(GameObject trackedObject, CameraZone zone)
         return trackedObjectZones.ContainsKey(trackedObject) && trackedObjectZones[trackedObject].Contains(zone);
     }
 
+    public List<CameraZone> GetCharacterCurrentZones(GameObject gameObject)
+    {
+        return trackedObjectZones[gameObject];
```

Comment:
I'd do a check here to make sure the key exists, maybe returning null otherwise (empty list may be confused for a valid object that just happens to be in no zones, so having that distinction would be good)